### PR TITLE
docs: build internal linking between glossary main and glossary detail page

### PR DESCRIPTION
**Pull request by tianzhou**
CleanShot 2022-04-20 at 23 43 16@2x

CleanShot 2022-04-20 at 23 42 37

### CODE REVIEW

File: pages/database-glossary/_slug/index.vue [hunk 1]
```diff
@@ -1,26 +1,92 @@
...
+      <div class="justify-between flex">
+        <div
+          v-for="(tag, tagIndex) in glossary.tagList"
+          :key="tagIndex"
+          class="inline-flex"
         >
-          {{ tag }}
-        </span>
+          <span
+            class="items-center px-3 py-1.5 mr-2 rounded-full text-base font-medium"
```
Comment (boojack) [inline pages/database-glossary/_slug/index.vue:51]: Do we need `v-if="getTagStyle(tag)"` right here?

Comment (tianzhou) [reply pages/database-glossary/_slug/index.vue]: Done

**Review by tianzhou — COMMENTED** [review]
PTAL

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**